lib/shopify-cli/project.rb

```diff
@@ -55,10 +55,11 @@ def has_current?
       #
       # #### Example
       #
-      #   type = ShopifyCli::Project.current_app_type
+      #   type = ShopifyCli::Project.current_project_type
       #
       def current_project_type
         return unless has_current?
+        return current.config['project_type'].to_sym if current.config.key?('project_type')
         current.config['app_type'].to_sym
```

Comment:
Agreed, we are *writing* only `project_type` from now on, but *reading* `project_type` first and falling back to `app_type`, as per this method.

Reply:
Right, I re-read this afterwards, so this works!!

I wonder if it would be better to filter this on the input, such that there's never going to be an `app_type` key stored, i.e., if after reading the yaml file results in an `app_type` key, copy the value to `project_type` and delete the `app_type` key.

That way, it's obvious in the rest of the code that the expected, _proper_ way is `project_type`.

Reply:
That's a valid point. We probably won't re-write the configuration once we read it, but if we ever do we'll get the added bonus of fixing up the configuration.

I'll change this so we replace app_type with project_type when we actually load the configs from the YAML file, rather than when we check the project type.